web_action_conditionable/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Web Action Conditionable
+========================
+
+This module was written to extend the functionality of actions in tree views.
+Odoo by default support:
+
+::
+
+   <tree delete="false" create="false">
+
+with this module you can:
+
+::
+   
+   <tree delete="state=='draft'">
+or ::
+
+   <tree create="state in ('draft', 'confirm')">
+
+It works in any tree view, so you can use it in One2many.
+
+
+Installation
+============
+
+To install this module, just follow basic steps to install an odoo module.
+
+
+Configuration
+=============
+
+Not needed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/web/7.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "7.0" for example
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/web/issues/new?body=module:web_action_conditionableversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```

Comment:
The link should be https://github.com/OCA/web/issues/new?body=module:web_action_conditionableversion:%207.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**